src/utils/_variables.scss

```diff
@@ -0,0 +1,16 @@
+$day-size: 100px;
+$day-count: 31;
+$border-size : $day-size / 100;
+$gap-size: $day-size / 100;
+$mockup-padding: $day-size / 10;
+$calendar-height: 100vh;
+$mockup-size: ($gap-size * 6) + (7 * $day-size);
+$start-shift: $day-size + $gap-size;
+$week-days:
+  "mon" 0,
+  "tue" $start-shift,
+  "wed" $start-shift * 2,
+  "thu" $start-shift * 3,
+  "fri" $start-shift * 4,
+  "sat" $start-shift * 5,
+  "sun" $start-shift * 6;
```

Comment:
Why not to use just an index instead of repeating `$start-shift`

Reply:
Good advice ! Absolutely agree . The way of calculation could be simplified.